randovania/data/json_data/prime3.txt

```diff
@@ -4141,18 +5057,23 @@ Asset id: 8444926274755356434
   > Dock to Defense Access
       Suit Type ≥ 4
   > Dock to Transit Station 1104
-      Suit Type ≥ 4
+      Suit Type ≥ 4 and After Skyway Access Wall
+  > Event - Open Wall
+      After Command Station Disable Defense
 
 > Dock to Defense Access; Heals? False
   * Morph Ball Door to Defense Access/Dock to Skyway Access
-  > Dock to Courtyard Passage
-      Suit Type ≥ 4
 
 > Dock to Transit Station 1104; Heals? False
   * Normal Door to Transit Station 1104/Dock to Skyway Access
   > Dock to Courtyard Passage
       Suit Type ≥ 4
 
+> Event - Open Wall; Heals? False
+  * Event Skyway Access Wall
+  > Dock to Courtyard Passage
+      Trivial
+
 ----------------
 Transit Station 3-B
 Leads to Research Facility
```

Comment:
- Add node "Tram to Research Facility - Transit Station 3-A" and connect to door as Trivial. Make this a teleporter node.
- Connect door to tram as trivial